spython/main/parse/docker.py

```diff
@@ -65,7 +65,11 @@ def _from(self, line):
            line: the line from the recipe file to parse for FROM
 
         '''
-        self.fromHeader = self._setup('FROM', line)
+        fromHeader = self._setup('FROM', line)
+
+        # Singularity does not support AS level
+        self.fromHeader = re.sub("AS level", "", fromHeader, flags=re.I)
```

Comment:
An unfortunate autocorrect in the original issue report.  This was intended to be 'AS devel', but the name can be anything.  So the regex should be something like "AS \S+"...

Reply:
ah my mistake! I should have looked at the docs first!